x-pack/plugins/endpoint/public/applications/endpoint/view/policy/policy_list.tsx

```diff
@@ -128,67 +120,52 @@ export const PolicyList = React.memo(() => {
         truncateText: true,
       },
       {
-        field: 'total',
-        name: i18n.translate('xpack.endpoint.policyList.totalField', {
-          defaultMessage: 'Total',
+        field: 'revision',
+        name: i18n.translate('xpack.endpoint.policyList.revisionField', {
+          defaultMessage: 'Revision',
         }),
-        render: renderFormattedNumber,
         dataType: 'number',
-        truncateText: true,
-        width: '15ch',
-      },
-      {
-        field: 'pending',
-        name: i18n.translate('xpack.endpoint.policyList.pendingField', {
-          defaultMessage: 'Pending',
-        }),
-        render: renderFormattedNumber,
-        dataType: 'number',
-        truncateText: true,
-        width: '15ch',
       },
       {
-        field: 'failed',
-        name: i18n.translate('xpack.endpoint.policyList.failedField', {
-          defaultMessage: 'Failed',
+        field: 'package',
+        name: i18n.translate('xpack.endpoint.policyList.versionField', {
+          defaultMessage: 'Version',
         }),
-        render: renderFormattedNumber,
-        dataType: 'number',
-        truncateText: true,
-        width: '15ch',
-      },
-      {
-        field: 'created_by',
-        name: i18n.translate('xpack.endpoint.policyList.createdByField', {
-          defaultMessage: 'Created By',
-        }),
-        truncateText: true,
-      },
-      {
-        field: 'created',
-        name: i18n.translate('xpack.endpoint.policyList.createdField', {
-          defaultMessage: 'Created',
-        }),
-        render: renderDate,
-        truncateText: true,
+        render(pkg) {
+          return `${pkg.title}  v${pkg.version}`;
+        },
       },
       {
-        field: 'updated_by',
-        name: i18n.translate('xpack.endpoint.policyList.updatedByField', {
-          defaultMessage: 'Last Updated By',
+        field: 'description',
+        name: i18n.translate('xpack.endpoint.policyList.descriptionField', {
+          defaultMessage: 'Description',
         }),
         truncateText: true,
       },
       {
-        field: 'updated',
-        name: i18n.translate('xpack.endpoint.policyList.updatedField', {
-          defaultMessage: 'Last Updated',
+        field: 'config_id',
+        name: i18n.translate('xpack.endpoint.policyList.agentConfigField', {
+          defaultMessage: 'Agent Configuration',
         }),
-        render: renderDate,
-        truncateText: true,
+        render(version: string) {
+          return (
+            // eslint-disable-next-line @elastic/eui/href-or-on-click
+            <EuiLink
+              href={`${services.application.getUrlForApp('ingestManager')}#/configs/${version}`}
+              onClick={(ev: SyntheticEvent) => {
```

Comment:
What is the reason for having the onClick here?

Reply:
It prevents a full browser refresh. 

This might be avoidable - a few weeks ago one of the updates from the kibana-platform team was a change required by Plugins for how to use react-router. Mainly, I think they are now passing down to us an instance of their `history` which we should use in initializing our router (or just use react-router's sub-routes - maybe?). If we do use their `history` instance, then doing getting the history object from react-router (using the hook) and then doing a `history.push` might do the same thing as this code below. 

I don't recall if we have an issue open to look into that routing change.

Reply:
BTW - @oatkiller question: I pinged the EUI team on why the `@elastic/eui/href-or-on-click` was enable, especially since we are mostly all using a router with history push. They said it was ok to turn it off. My question: should we turn it off across our plugin? (by introducing a `.eslingrc.js` file)

Reply:
Gotcha. Looking through the code I see a few places that are using a similar pattern. 

I wrote this up a while ago, but would like your thoughts: https://github.com/oatkiller/kibana/commit/d6861d858a432f43d2fa48792beaa79f3ef43e23

Seeing that you're doing `navigateToApp`, instead of `history.push`, I don't think the commit would work for you out of the box. But I would like to have an easy way to reuse the logic that guards `event.preventDefault`

also @parkiino  cause I was talking to her as well.

Reply:
re : changing lint rules. I've been contemplating changing lint rules since day 0. @stacey-gammon argued that having consistent rules across all of Kibana is worthwhile because it allows developers to contribute across the app with the same standards (and allows code to be theoretically copy-pasted around the app as well.)

In the past I used lint rules (and other forms of static analysis) heavily as a way to enforce consistency across the app, so it's something I still think about.

With that in mind, I think we should try to argue for the lint rules changes to happen Kibana-wide. If Kibana contributors at large disagree'd with our change, we could reevaluate then.

Reply:
Yeah, I was holding on to a reference to that generic `onClick` handler in order to revisit it soon. (actually: @kevinlog I thought you opened a separate issue to track doing that, but I can't find it)
I like the idea and concept and I think that using a hooks will make the DX really pleasant. Actually, maybe a hook to also do a generic `onClick` handler for `navigateToApp` would be nice. I'll check to see if we have a separate issue opened to track it. if not, I'll create one and will reference your code sample :)

Reply:
>  I think we should try to argue for the lint rules changes to happen Kibana-wide. If Kibana contributors at large disagree'd with our change, we could reevaluate then.

💯 

For this particular issue, I think this should be a moot point once everything is converted to NP, there will no longer be the full page refresh navigating among apps. Though I'm surprised using `navigateToApp` somehow gets around the full page refresh.  So maybe I am missing something else.  cc @joshdover

Reply:
Maybe we're hitting the issue because we have not implemented the recent change in the NP to use the `history` instance provided by plugin `mount()` (this change #56705 )???
Will try that out to see if that is it.

Reply:
Ok, I tried it by using `history.push()` instead of `navigateToApp()` after making changes in our router to use the `history` instance passed by `mount()`, but still got full page refresh. Perhaps I'm missing something as to how we should interop with the core top-level router ( @joshdover ?? 😃 )

I'm laving in the code that uses `navigateToApp()` for now and will refactor if there is a better pattern to follow.

Reply:
@stacey-gammon no problem at all - appreciate the feedback.
Good to know that the platform team has this on their list of items to address in the future. The ability to "just handle it" is exactly what @chandlerprall suggested on Slack when I first approach the EUI team about possibly turning this ESLint rule off - 

> "...I recommend having an onClick at the document.body level that can programmatically interact with the router...."